api/yelp_beans/templates/match_email.html

```diff
@@ -4,8 +4,8 @@
     <td>
         <p>Hey {{user.first_name}}! </p>
         <p>
-            Congrats! We've matched you this week for Yelp Beans ({{meeting_title}}) on {{meeting_start_day}}
-            from {{meeting_start_time}} to {{meeting_end_time}}.Here's who you are meeting:
+            Congrats! We've matched you this week for Yelp Beans ({{meeting_title}})! Feel free to talk to your match about when you'd like to meet, but we suggest meeting on {{meeting_start_day}}
+            from {{meeting_start_time}} to {{meeting_end_time}}. Here's who you are meeting:
```

Comment:
```suggestion
            Congrats! We've matched you this week for Yelp Beans ({{meeting_title}})! Feel free to talk to your match about when
            you'd like to meet, but we suggest meeting on {{meeting_start_day}} from {{meeting_start_time}} to
            {{meeting_end_time}}. Here's who you are meeting:
```

nit: Should we stick to a max line length similar to what we do in python files (131 characters)? It looks like the we are keeping to a length close to this in other places in html files.